packages/tokens/figma/dark.json

```diff
@@ -151,10 +151,6 @@
     "$value": "#e5a000",
     "$type": "color"
   },
-  "uswds-system-color-gold-vivid-40": {
```

Comment:
Were removing this and `uswds-system-color-green-cool-vivid-30` here and the light figma file intentional or did it happen with timing around Narin's branch being merged?

Reply:
Good catch! It didn't look like that in VS Code, but Narin was just able to help me fix it here. Thanks!